manim/scene/scene_file_writer.py

```diff
@@ -386,11 +387,11 @@ def finish(self):
         If save_last_frame is True, saves the last
         frame in the default image directory.
         """
-        if file_writer_config["write_to_movie"]:
+        if config["write_to_movie"]:
             if hasattr(self, "writing_process"):
                 self.writing_process.terminate()
             self.combine_movie_files()
-            if file_writer_config["flush_cache"]:
+            if config["flush_cache"]:
                 self.flush_cache_directory()
             else:
                 self.clean_cache()
```

Comment:
**Question** This used to be there
```py
        if config["save_last_frame"]:
            self.scene.update_frame(ignore_skipping=True)
```
What happened? Where/why did you move it ? 
I saw on a precedent commit that you changed `ignore_skipping` to False, which is IMO not a good idea, but I can't even see theses lines on HEAD lol

Reply:
This was moved to `Renderer.finish()`, by a previous PR, not this one.